hsia/hsia_tasks.py

```diff
@@ -0,0 +1,209 @@
+import os
+import shutil
+import glob
+import netrc
+import subprocess
+import tarfile
+import numpy as np
+import pandas as pd
+import geopandas as gpd
```

Comment:
Looks like maybe some unused imports?

```suggestion
```

Reply:
You're right! These were added previously when I had different planned path for generating the GeoTIFFs and are now removed. Good eye!